packages/patternfly-4/react-core/src/components/Alert/Alert.tsx

```diff
@@ -71,6 +71,7 @@ const Alert: React.FunctionComponent<AlertProps & InjectedOuiaProps> = ({
       aria-label={ariaLabel}
       {...(ouiaContext.isOuia && {
         'data-ouia-component-type': 'Alert',
+        'data-ouia-component-ns': 'PF4',
```

Comment:
Could this be defined in one place, instead of within each component?

Reply:
@dlabrecq it's a good point but my knowledge of typescript is limited, I believe it can be done somewhere here https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/withOuia/withOuia.tsx

Reply:
Thinking we just need to import a common file. Perhaps `ouia.ts` or `withOuia.tsx` would be a good location to store the namespace?

.../packages/patternfly-4/react-core/src/components/withOuia/ouia.ts

Reply:
Any chance we can use a qualified URI as the namespace, instead of a basic string? Something like `https://www.patternfly.org/v4` as the namespace is more semantic and readable.

Reply:
@psav @karelhala ^

Reply:
Here's a good article in case it is helpful

https://www.w3.org/2001/tag/doc/URNsAndRegistries-50-2006-08-17.html#args_http